charts/alfresco-search-service/charts/alfresco-insight-zeppelin/Chart.yaml

```diff
@@ -11,8 +11,8 @@ name: alfresco-insight-zeppelin
 version: 3.0.1
```

Comment:
I don't know what we should do with this (keep in sync with parent?)

Reply:
I will just bump it independently. There very few chance it ever evolves without search service but that's still a different code base where bumps should be independent (e.g. a bug fix in one of the chart themselves)